torch/onnx/utils.py

```diff
@@ -128,7 +141,7 @@ def export(model, args, f, export_params=True, verbose=False, training=False,
     _export(model, args, f, export_params, verbose, training, input_names, output_names,
             operator_export_type=operator_export_type, opset_version=opset_version,
             _retain_param_name=_retain_param_name, do_constant_folding=do_constant_folding,
-            strip_doc_string=strip_doc_string)
+            strip_doc_string=strip_doc_string, example_outputs=example_outputs, dynamic_axes=dynamic_axes)
```

Comment:
please don't fix the example_outputs in diff, it's already fixed in another ongoing one.

Reply:
Sure.